pydummy

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
I don't normally like this pattern, which seems very common in Python scripts generally. I do rather prefer to know exactly which Python interpreter the script is going to run under without referring to `PATH`. That being said, this script is there as an example to RA developers and not for user consumption, so there's no harm in making this change. Ack.